packages/react-tinacms-inline/src/blocks/inline-block-field-controls.tsx

```diff
@@ -67,9 +67,17 @@ export function BlocksControls({
   const blockMoveDownRef = React.useRef<HTMLButtonElement>(null)
 
   const addBeforePosition =
-    direction === 'column' ? 'top' : direction === 'row' ? 'left' : 'top'
+    direction === 'vertical'
+      ? 'top'
+      : direction === 'horizontal'
+      ? 'left'
+      : 'top'
```

Comment:
I think this can be simplified
```suggestion
    direction === 'horizontal'
      ? 'left'
      : 'top'
```